### ISSUE TITLE
Unable to install CURL `No such file or directory - lib/mk-ca-bundle.pl`

### ISSUE THREAD

**Issue by tachigami** [cloudflare/homebrew-cloudflare#45]
```shell
brew install --formula --HEAD -s curl.rb
==> Cloning https://github.com/cloudflare/quiche.git
Updating /Users/bronner/Library/Caches/Homebrew/curl--quiche--git
==> Checking out branch master
Already on 'master'
Your branch is up to date with 'origin/master'.
HEAD is now at 9dfeaaf examples: use mio-v0.8
Entering 'quiche/deps/boringssl'
/Users/bronner/Library/Caches/Homebrew/curl--quiche--git/quiche/deps/boringssl
==> Cloning https://github.com/curl/curl.git
Updating /Users/bronner/Library/Caches/Homebrew/curl--git
==> Checking out branch master
Already on 'master'
Your branch is up to date with 'origin/master'.
HEAD is now at c2610db63 headers.h: make Curl_headers_push() be CURLE_OK when not built
==> cargo build --release --package=quiche --features=ffi,pkg-config-meta,qlog
==> ./buildconf
==> ./configure --prefix=/usr/local/Cellar/curl/HEAD-c2610db --with-ssl=/private/tmp/curl-20220324-21393-6jfx5j/quiche/quiche/deps/boringssl/src --with-ca-fallback --with-secure-transport --with-default-ssl-backend=op
==> make install
==> make install -C scripts
Error: An exception occurred within a child process:
  Errno::ENOENT: No such file or directory - lib/mk-ca-bundle.pl
```

MacOS 12.3

What am I doing wrong?

**Comment by love4taylor** [comment]
```
libexec.install "scripts/mk-ca-bundle.pl"
```

**Comment by TeroBlaZe** [comment]
> ```
> libexec.install "scripts/mk-ca-bundle.pl"
> ```

Thanks. It worked!

Why didn't they change it in the repo?

**Comment by ryankirkman** [comment]
Thanks for the pointer! I submitted a PR with this fix: https://github.com/cloudflare/homebrew-cloudflare/pull/46

You can see more context in the PR about what changed:

> In curl/curl#8625, `mk-ca-bundle.pl` was moved from `lib/` to `scripts/`

**Comment by cujof** [comment]
PR https://github.com/cloudflare/homebrew-cloudflare/pull/51 checks for both lib/mk-ca-bundle.pl and scripts/mk-ca-bundle.pl and executes whichever exists.

**Comment by junhochoi** [comment]
I opened #62 for build fix with latest, basically converting #51's script.
